rent_golem/backend/rent_golem/accounts/urls.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework.urls import path
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView,
+)
+
+urlpatterns = [
+    path('api/token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
+    path('api/token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
```

Comment:
I think we need to remove `api/` prefix. Right now, full path is probably `/api/v1/accounts/api/token`. Please check afterwards with `python manage.py show_urls`